### PR TITLE
Reformat to import at the item level.

### PR DESCRIPTION
**Pull request by bnaecker**

### CODE REVIEW

**Comment by bnaecker** [comment]
Resolves #258

File: opte/src/engine/arp.rs [hunk 1]
```diff
@@ -11,23 +11,65 @@
...
+/// * RFC 826 -- An Ethernet Address Resolution Protocol
+use super::ether::ETHER_TYPE_ARP;
+/// Address Resolution Protocol
+///
+/// Relevant Docs
+///
+/// * RFD 9 -- NETWORKING CONSIDERATIONS
+/// ** §1.13 ARP
+/// * RFC 826 -- An Ethernet Address Resolution Protocol
+use super::ether::ETHER_TYPE_IPV4;
```
[the remark this answers is not in the document]
Comment (bnaecker) [reply opte/src/engine/arp.rs:38]: Oh, I saw that but thought it was intentional. I'll rework the comment manually, thanks.

File: opte/src/engine/arp.rs [hunk 2]
```diff
@@ -11,23 +11,65 @@
...
+/// * RFD 9 -- NETWORKING CONSIDERATIONS
+/// ** §1.13 ARP
+/// * RFC 826 -- An Ethernet Address Resolution Protocol
+/// Address Resolution Protocol
+///
+/// Relevant Docs
+///
+/// * RFD 9 -- NETWORKING CONSIDERATIONS
+/// ** §1.13 ARP
+/// * RFC 826 -- An Ethernet Address Resolution Protocol
```
Comment (rzezeski) [inline opte/src/engine/arp.rs:34]: This is still all messed up. At the top of the file, it should have just been:

```
/ Address Resolution Protocol
 ///
 /// Relevant Docs
 ///
 /// * RFD 9 -- NETWORKING CONSIDERATIONS
 /// ** §1.13 ARP
 /// * RFC 826 -- An Ethernet Address Resolution Protocol
```

(Yes, the coment is kind of crap, but I'm just trying to point out fmt has gone off the rails here).

Comment (bnaecker) [reply opte/src/engine/arp.rs:34]: Weird, thanks. I'll look again.

Comment (bnaecker) [reply opte/src/engine/arp.rs:34]: Oh I think I see what happened. That comment was attached to a nested import, and `cargo fmt` must have duplicated it when it expanded each item to its own line. I'll remove the extra copies.

**Comment by bnaecker** [comment]
It also looks like `cargo fmt` decided to leave braces around self-imports, e.g., `use super::ether::{self}`. I'll fix those manually too.

**Review by rzezeski — APPROVED** [review]

**Pull request merged**